### PR TITLE
console: fix checkbox for forwarding client headers in actions (close #4595)

### LINKED ISSUE #4595: Strange behavior for "forward client headers..." checkbox in action
Hello. The checkbox cannot be marked clicking directly on it but works clicking on the label. Also looks checked when switch between sync and async radio button. 

Look at the following gif

Thanks

demo

### PR DESCRIPTION
**Pull request by lastnamearya**
### Changelog

- [X] `CHANGELOG.md` is updated with user-facing content relevant to this PR.

### Affected components

- [X] Console

### Related Issues

#4595 

#### Breaking changes

- [X] No Breaking changes

### CODE REVIEW

File: console/src/components/Services/Actions/Common/components/HeaderConfEditor.js [hunk 1]
```diff
@@ -32,15 +32,12 @@ const HandlerEditor = ({
         />
       </h2>
       <div className={`${styles.add_mar_bottom_mid}`}>
-        <label
-          className={`${styles.add_mar_right} ${styles.cursorPointer}`}
-          onClick={toggleForwardClientHeaders}
-        >
+        <label className={styles.add_mar_right}>
```
Comment (wawhal) [inline console/src/components/Services/Actions/Common/components/HeaderConfEditor.js:35]: Can you make the entire label (along with the checkbox) `cursor:pointer`?

Comment (lastnamearya) [reply console/src/components/Services/Actions/Common/components/HeaderConfEditor.js:35]: Sure @wawhal

Comment (lastnamearya) [reply console/src/components/Services/Actions/Common/components/HeaderConfEditor.js]: pushed

File: CHANGELOG.md [hunk 2]
```diff
@@ -39,6 +40,7 @@ Read more about the session argument for computed fields in the [docs](https://h
 - console: avoid count queries for large tables (#4692)
 - console: add read replica support section to pro popup (#4118)
 - console: allow modifying default value for PK (fix #4075) (#4679)
+- console: fix checkbox for actions headerconfeditor (#4595)
```
Comment (tirumaraiselvan) [inline CHANGELOG.md:43]:
```suggestion
- console: fix checkbox for forwarding client headers in actions (#4595)
```

Comment (lastnamearya) [reply CHANGELOG.md]: updated @tirumaraiselvan :)

**Pull request merged**